src/components/Professional/style.js

```diff
@@ -0,0 +1,51 @@
+import { css } from "emotion"
+
+export const wrapper = css`
+  margin-top: 24px;
+  @media (min-width: 81.25em) {
+    margin-bottom: 96px;
+  }
+`
+export const box = css`
+  background-color: var(--primary-10);
+  padding: 24px 12px;
+  border-radius: 6px;
+  display: flex;
+  font-size: 25.92px;
+  font-size: 1.62rem;
```

Comment:
use only one font-size property. The first one is always ignored in favor of the latter.